keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -44,10 +44,12 @@ const (
 	CentOSType     = "centos"
 
 	KubeEdgeDownloadURL          = "https://github.com/kubeedge/kubeedge/releases/download"
+	EdgecoreServiceFileURL       = "https://raw.githubusercontent.com/kubeedge/kubeedge/release-%s/build/tools/%s"
```

Comment:
```suggestion
	EdgeCoreServiceFileURL       = "https://raw.githubusercontent.com/kubeedge/kubeedge/release-%s/build/tools/%s"
```

Reply:
I find these two line looking the same, am I missing anything here?

Reply:
Bad diff? I can only see it is a new line added here.

Reply:
fixed